Content.Shared/Actions/ActionContainerSystem.cs

```diff
@@ -26,6 +30,25 @@ public override void Initialize()
         SubscribeLocalEvent<ActionsContainerComponent, ComponentShutdown>(OnShutdown);
         SubscribeLocalEvent<ActionsContainerComponent, EntRemovedFromContainerMessage>(OnEntityRemoved);
         SubscribeLocalEvent<ActionsContainerComponent, EntInsertedIntoContainerMessage>(OnEntityInserted);
+        SubscribeLocalEvent<ActionsContainerComponent, MindAddedMessage>(OnMindAdded);
+        SubscribeLocalEvent<ActionsContainerComponent, MindRemovedMessage>(OnMindRemoved);
+    }
+
+    private void OnMindAdded(EntityUid uid, ActionsContainerComponent component, MindAddedMessage args)
+    {
+        if(!_mind.TryGetMind(uid, out var mindId, out _))
+            return;
+
+        if (!TryComp<ActionsContainerComponent>(mindId, out var mindActionContainerComp))
+            return;
+
+        if (!HasComp<GhostComponent>(uid) && mindActionContainerComp.Container.ContainedEntities.Count > 0 )
+            _actions.GrantContainedActions(uid, mindId);
```

Comment:
This seems somewhat skrunkly, but I also agree that it's probably confusing if things like spells appear as available to ghosts despite them not being able to use them.

It's fine for now but long term there's probably some better way of doing this

Reply:
I guess it could be something like "AllowGhost" somewhere and then just grant that action. When it comes around we'll figure something out.